src/_lib/modules/email-input/email-input.component.ts

```diff
@@ -19,14 +19,15 @@ import { Subscription } from 'rxjs';
                 <button class="close" (click)="removeEmail(email)">×</button>
             </div>
 
-            <div class="input-container">
+            <div class="input-container" [class.persistent-placeholder]="!!persistentPlaceholder">
```

Comment:
I think !! will cause a bit of confusion here. If you leave it as just persistentPlaceholder without the !! it will do the job as well. An empty string is a false value and if its not empty it will become true.